client/python/armada_client/client.py

```diff
@@ -32,14 +30,10 @@ class ArmadaClient:
     :param channel: gRPC channel used for authentication. See
                     https://grpc.github.io/grpc/python/grpc.html
                     for more information.
-    :param max_workers: number of cores for thread pools, if unset, defaults
-                        to number of CPUs
     :return: an Armada client instance
     """
 
-    def __init__(self, channel, max_workers: Optional[int] = None):
-        self.executor = ThreadPoolExecutor(max_workers=max_workers or os.cpu_count())
-
+    def __init__(self, channel):
```

Comment:
Why did we make this change?

Reply:
I don't really know why it was there in the first place but curious why it was removed.

Reply:
`self.executor` is not/never used, therefore neither is `max_workers`.